wallet/src/Pos/Wallet/Web/Methods/Restore.hs

```diff
@@ -63,24 +69,42 @@ newWalletFromBackupPhrase passphrase CWalletInit {..} isReady = do
 
     return (skey, cAddr)
 
-newWallet :: L.MonadWalletLogic ctx m => PassPhrase -> CWalletInit -> m CWallet
-newWallet passphrase cwInit = do
+newWalletHandler :: L.MonadWalletLogic ctx m => PassPhrase -> CWalletInit -> m CWallet
+newWalletHandler passphrase cwInit = do
     -- A brand new wallet doesn't need any syncing, so we mark isReady=True
-    (_, wId) <- newWalletFromBackupPhrase passphrase cwInit True
+    (_, wId) <- newWallet passphrase cwInit True
     removeHistoryCache wId
     -- BListener checks current syncTip before applying update,
     -- thus setting it up to date manually here
     withStateLockNoMetrics HighPriority $ \tip -> setWalletSyncTip wId tip
     L.getWallet wId
 
-restoreWallet :: L.MonadWalletLogic ctx m => PassPhrase -> CWalletInit -> m CWallet
-restoreWallet passphrase cwInit = do
-    -- Restoring a wallet may take a long time.
-    -- Hence we mark the wallet as "not ready" until `syncWalletOnImport` completes.
-    (sk, wId) <- newWalletFromBackupPhrase passphrase cwInit False
-    -- `syncWalletOnImport` automatically marks a wallet as "ready".
-    syncWalletOnImport sk
```

Comment:
Maybe I'm missing it, but where synchronization is initiated now?

Reply:
@Martoon-00 This is a set of ongoing patches, that's why I'm merging into `feature/CBR-90` 😛 

This one has no sync, it will be added in a separate PR.

Reply:
Ah okay :slightly_smiling_face: